lib/tasks/rtd_location_create.rake

```diff
@@ -0,0 +1,14 @@
+require 'yaml'
+
```

Comment:
This is new! What does it do?

Reply:
Actually we can delete that. It reads the application.yml file so the rake task knows about my API keys, but we aren't' doing that in our rails app.  It is habit for me to include this at the top of my rake tasks so I'm not confused when my rake does not work.